src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMessageHandler.java

```diff
@@ -105,9 +108,28 @@ private void logSentFor(final Interchange interchange) {
     private void logRecepSentFor(final Interchange interchange) {
```

Comment:
```suggestion
    private void logNhsackSentFor(final Interchange interchange) {
```

To remove all references to Receps